gradescope_utils/autograder_utils/json_test_runner.py

```diff
@@ -65,31 +65,43 @@ def getOutput(self):
             return out
 
     def buildResult(self, test, err=None):
-        passed = (err == None)
-
+        failed = err is not None
         weight = self.getWeight(test)
         tags = self.getTags(test)
         number = self.getNumber(test)
         visibility = self.getVisibility(test)
         hide_errors_message = self.getHideErrors(test)
         score = self.getScore(test)
-        if score is None:
-            score = weight if passed else 0.0
-
         output = self.getOutput()
         if err:
             if hide_errors_message:
                 output += hide_errors_message
             else:
-                output += "Test Failed: {0}\n".format(err[1])
+                if output:
+                    # create a double newline
+                    if output.endswith('\n'):
+                        output += '\n'
+                    else:
+                        output += '\n\n'
+                output += f'Fail: {err[1] or "<no msg>"}\n'
```

Comment:
Was looking at https://github.com/gradescope/gradescope-utils/issues/23 as well and it seems like we could resolve that here pretty easily:

```suggestion
                else:
                    output = ""
                output += f'Fail: {err[1] or "<no msg>"}\n'
```